postgresql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -55,3 +55,56 @@ datatypesSchema=[{"key":"id","value":"string"},{"key":"col1","value":"string"},{
   {"key":"col52","value":"string"},{"key":"col53","value":"string"},{"key":"col54","value":"string"},\
   {"key":"col55","value":"string"},{"key":"col56","value":"string"},{"key":"col57","value":"string"},\
   {"key":"col58","value":"int"},{"key":"col59","value":"int"},{"key":"col60","value":"long"}]
+
+#POSTGRESQL Invalid Properties
+invalidRef=invalidRef&^*&&*
+invalidDatabaseName=invalidDB
+invalidImportQuery=select
+invalidTableNameImportQuery=select * from abc;
+invalidDriverName=abcdriver
+invalidHost=testhost
+invalidPort=testPort
+invalidUserName=testUser
+invalidPassword=testPassword
+invalidBoundingQuery=SELECT MIN(id),MAX(id) FROM table
+invalidTable=table
+
+#POSTGRESQL Valid Properties
+connectionArgumentsList=[{"key":"queryTimeout","value":"-1"}]
+connectionTimeout=150
+numberOfSplits=2
+zeroValue=0
+splitByColumn=ID
+postgreImportQuery=select * from public.SOURCETABLE_dnbbbcpszj where $CONDITIONS
+
+#bq properties
+projectId=cdf-athena
+dataset=test_automation
+bqOutputMultipleDatatypesSchema= [{"key":"col1","value":"bytes"},{"key":"col2","value":"string"},\
+  {"key":"col3","value":"date"},{"key":"col4","value":"double"},{"key":"col5","value":"decimal"},\
+  {"key":"col6","value":"timestamp"},{"key":"col7","value":"decimal"},{"key":"col8","value":"boolean"},\
+  {"key":"col9","value":"long"},{"key":"col10","value":"time"}]
+jdbcUrl = jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=%s;OAuthType=3;
```

Comment:
Don't need this jdbcUrl filed we will not use this as we are not using the previous data validation logic.

Reply:
Removed.